routing_common/routing_common_tests/vehicle_model_test.cpp

```diff
@@ -112,98 +110,81 @@ void CheckSpeed(initializer_list<uint32_t> const & types, InOutCitySpeedKMpH con
               SpeedKMpH(150.0 /* weight */, 150.0 /* eta */) /* out of city */});
-  CheckSpeed({GetType("highway", "primary")}, {SpeedKMpH(90.0, 90.0), SpeedKMpH(120.0, 120.0)});
-  CheckSpeed({GetType("highway", "residential")}, {SpeedKMpH(22.5, 27.5), SpeedKMpH(25.0, 30.0)});
+  CheckSpeed({primary}, {SpeedKMpH(90.0, 90.0), SpeedKMpH(120.0, 120.0)});
+  CheckSpeed({residential}, {SpeedKMpH(22.5, 27.5), SpeedKMpH(25.0, 30.0)});
 }
 
-UNIT_CLASS_TEST(VehicleModelTest, VehicleModel_Speed_MultiTypes)
+UNIT_CLASS_TEST(VehicleModelTest, Speed_MultiTypes)
 {
-  uint32_t const typeTunnel = GetType("highway", "secondary", "tunnel");
-  uint32_t const typeSecondary = GetType("highway", "secondary");
-  uint32_t const typeHighway = GetType("highway");
+  uint32_t const typeHighway = classif().GetTypeByPath({"highway"});
 
-  CheckSpeed({typeTunnel, typeSecondary}, kDefaultSpeeds.Get(HighwayType::HighwaySecondary));
-  CheckSpeed({typeTunnel, typeHighway}, kDefaultSpeeds.Get(HighwayType::HighwaySecondary));
-  CheckSpeed({typeHighway, typeTunnel}, kDefaultSpeeds.Get(HighwayType::HighwaySecondary));
+  CheckSpeed({secondaryTunnel, secondary}, kDefaultSpeeds.Get(HighwayType::HighwaySecondary));
+  CheckSpeed({secondaryTunnel, typeHighway}, kDefaultSpeeds.Get(HighwayType::HighwaySecondary));
+  CheckSpeed({typeHighway, secondaryTunnel}, kDefaultSpeeds.Get(HighwayType::HighwaySecondary));
 }
 
-UNIT_CLASS_TEST(VehicleModelTest, VehicleModel_OneWay)
+UNIT_CLASS_TEST(VehicleModelTest, OneWay)
 {
-  uint32_t const typeBridge = GetType("highway", "secondary", "bridge");
-  uint32_t const typeOneway = GetOnewayType();
+  CheckSpeed({secondaryBridge, oneway}, kDefaultSpeeds.Get(HighwayType::HighwaySecondary));
+  CheckOneWay({secondaryBridge, oneway}, true);
+  CheckSpeed({oneway, secondaryBridge}, kDefaultSpeeds.Get(HighwayType::HighwaySecondary));
+  CheckOneWay({oneway, secondaryBridge}, true);
 
-  CheckSpeed({typeBridge, typeOneway}, kDefaultSpeeds.Get(HighwayType::HighwaySecondary));
-  CheckOneWay({typeBridge, typeOneway}, true);
-  CheckSpeed({typeOneway, typeBridge}, kDefaultSpeeds.Get(HighwayType::HighwaySecondary));
-  CheckOneWay({typeOneway, typeBridge}, true);
-
-  CheckOneWay({typeOneway}, true);
+  CheckOneWay({oneway}, true);
 }
 
-UNIT_CLASS_TEST(VehicleModelTest, VehicleModel_DifferentSpeeds)
+UNIT_CLASS_TEST(VehicleModelTest, DifferentSpeeds)
 {
-  uint32_t const typeSecondary = GetType("highway", "secondary");
-  uint32_t const typePrimary = GetType("highway", "primary");
-  uint32_t const typeOneway = GetOnewayType();
-
-  CheckSpeed({typeSecondary, typePrimary}, kDefaultSpeeds.Get(HighwayType::HighwaySecondary));
-
-  CheckSpeed({typeSecondary, typePrimary, typeOneway}, kDefaultSpeeds.Get(HighwayType::HighwaySecondary));
-  CheckOneWay({typePrimary, typeOneway, typeSecondary}, true);
+  // Why to have a bit artificial test with multiple road types? Show that order is matter?
```

Comment:
По-русски что хотел сказать? Вот это?
Для чего нужен слегка искусственный тест с различными типами дорог? Показать, что важен порядок?

```suggestion
  // What is the purpose of a slightly artificial test with different types of roads? To show that order is important?
```